src/hiposfer/kamal/parsers/gtfs/preprocessor.clj

```diff
@@ -21,7 +21,7 @@
 ;; routes
 (s/def ::route_id spec/integer?)
 (s/def ::route_type spec/integer?)
-(s/def ::route (s/keys :req-un [::route_id ::gtfs/route_short_name ::route_type]
+(s/def ::route (s/keys :req-un [::route_id (or ::gtfs/route_short_name ::route_type)]
```

Comment:
By `route_type` you mean route long name?

Reply:
oh shit that is an error :D it should be long_name